sheetstorm/urls.py

```diff
@@ -14,7 +14,10 @@
 from django.contrib import admin
 from django.urls import include
 
+from users import views
+
 urlpatterns = [
+    url(r"^$", views.index, name="home"),
```

Comment:
I am just wondering, about changing this view to TemplateView because  we are using CBV, everywhere. However, maybe it is not needed.

Reply:
Good point. Changed :+1: